app/src/forms/common/utils.js

```diff
@@ -209,109 +210,98 @@ const checkIsFormExpired = (formSchedule = {}) => {
     message: '',
   };
 
-  if (formSchedule && formSchedule.enabled) {
-    //Check if Form open date is in past or Is form already started for submission
-    if (formSchedule.openSubmissionDateTime) {
-      let startDate = moment(formSchedule.openSubmissionDateTime).format('YYYY-MM-DD HH:MM:SS');
-      let closingDate = null;
-      if (formSchedule.scheduleType === 'closingDate' && formSchedule.closeSubmissionDateTime) {
-        closingDate = moment(formSchedule.closeSubmissionDateTime).format('YYYY-MM-DD HH:MM:SS');
-      }
-      let isFormStartedAlready = moment().diff(startDate, 'seconds'); //If a positive number it means form get started
-      if (isFormStartedAlready >= 0) {
-        //Form have valid past open date for scheduling so lets check for the next conditions
-        if (isFormStartedAlready && formSchedule.enabled && formSchedule.scheduleType !== 'manual') {
-          if (formSchedule.closingMessageEnabled) {
-            if (formSchedule.closingMessage) {
-              result = { ...result, message: formSchedule.closingMessage };
-            } else {
-              result = { ...result, message: 'Something went wrong.' };
-            }
-          } else {
-            result = { ...result, message: 'The form submission period has expired.' };
-          }
+  if (formSchedule && formSchedule.enabled && formSchedule.openSubmissionDateTime) {
+    // The start date is the date that the form should be scheduled to be open to allow submissions
+    let startDate = moment(formSchedule.openSubmissionDateTime).startOf('day');
```

Comment:
Instead of formatting the open date to just the date the user picked which didn't even include the time, we set it to the date the user picked at the start of the day.